spec/support/database_cleaner.rb

```diff
@@ -4,11 +4,13 @@
     DatabaseCleaner.clean_with(:truncation)
```

Comment:
well, that line isn't needed :)

before(:suite) works only once, like before(:all), but being called even before than before(:all)

before(:each) is being called before each running spec. so in your ```ruby config.before(:each)```
you reset strategy each time you check new spec.